api-reference/receipts/endpoints.markdown

```diff
@@ -0,0 +1,672 @@
+      "id": "df8c1161d917439c9e6f141fd0d6b588",
+      "image": "",
+      "imageId": "",
+      "receipt": {
+        ...Receipt JSON...
+      },
+      "userId": "08bdda1e-0d4f-4261-9f1b-f9b8d9f817d6",
+      "validationSchema": "http://schema.concursolutions.com/car-rental-receipt.schema.json",
+      "self": "https://us.api.concursolutions.com/receipts/v4/df8c1161d917439c9e6f141fd0d6b588",
+      "template": "https://us.api.concursolutions.com/receipts/v4/{receiptId}"
+    },
+    ...
+  ]
+}
+```
+
+[Back to Top](#endpoints)
+
+##### Endpoint: Get Receipt Image
+
+###### _GET /v4/:receiptId/image_
+
+|Parameter|Requirement|Value|
+|---|---|---|
+|receiptId|required|The UUID of the receipt associated with the image.|
+
+If an image or PDF document was generated by or POSTed to Receipts v4, this endpoint can return the image in the same format that it was originally received by the API. Images for receipts created with v3 of the API are _not_ accessible via this endpoint.
+
+_Example Requests:_
+
+cURL:
+
+```shell
+curl -H "Authorization: Bearer {YOUR ACCESS TOKEN}" https://us.api.concursolutions.com/receipts/v4/{RECEIPT ID}/image
+```
+
+HTTPie:
+
+```shell
+http https://us.api.concursolutions.com/receipts/v4/{RECEIPT ID}/image "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+[Back to Top](#endpoints)
+
+### Image-Only Receipts
+
+|Endpoint|Response Format|Request Summary|
+|---|---|---|
+|[POST /v4/users/:userId/image-only-receipts](#endpoint-post-an-image-only-receipt)|N/A|Post an image-only receipt|
+|[GET /v4/users/:userId/image-only-receipts](#endpoint-get-image-only-receipts-by-userid)|JSON|Get a user's image-only receipts|
+|[GET /v4/image-only-receipts/:receiptId](#endpoint-get-an-image-only-receipt-by-id)|JSON|Get an image-only receipt by ID|
+
+##### Endpoint: Post an Image-Only Receipt
+
+###### _POST /v4/users/:userId/image-only-receipts_
+
+|Parameter|Requirement|Value|
+|---|---|---|
+|userId|required|The UUID of the user to whom the receipt image belongs.|
+|image|required|Image of the receipt.|
+
+- Image constraints
+  - Image size must not exceed 5MB
+  - Image must be one of the supported file types: PNG, JPG/JPEG, TIFF/TIF, GIF, or PDF
+
+Successful POST requests will receive a response of 202 Accepted. The Location header of the response contains a URL for your receipt image. Once the receipt has been processed, it can be retrieved at this URL. The Link header of the response contains a processing-status URL for your receipt image.
+
+To post a receipt image, use multipart form data. The Content-Type:multipart/form-data header must be set. The image should be included under the key image.
+
+_Example Requests:_
+
+cURL:
+
+```shell
+curl -vk -X POST https://us.api.concursolutions.com/receipts/v4/users/{USER ID FROM YOUR ID TOKEN}/image-only-receipts \
+-H "Authorization: Bearer {YOUR ACCESS TOKEN}" \
+-H "Content-Type:multipart/form-data" \
+-F "image=@{PATH TO YOUR IMAGE};type=image/{FILE TYPE OF YOUR IMAGE}"
```

Comment:
I was just coping format, but I missed the MIME stuff, so I'm adding that. Does that help or am I misunderstanding your question?

Reply:
Yeah we should probably be specific that `image/` is required in the type.

Reply:
I think I might be confused here as well. @merittrompeter can you clarify your question?

Reply:
Sorry guys, again this is out of ignorance, there's probably a good reason.

Is there some reason there is an @ in image=@{PATH TO YOUR IMAGE} and not in type=image/{FILE TYPE OF YOUR IMAGE}?

Reply:
Yeah the @ makes it so that the file gets uploaded as the data part of the key/value pair image={some data}. The value for `type` is just a string, not a file

Reply:
Right, I figured it was something like that. Thanks! 👍